app/controllers/request-psp-test-account/get.controller.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+const { response, renderErrorView } = require('../../utils/response')
+const { NOT_STARTED, CREATED, REQUEST_SUBMITTED } = require('../../models/psp-test-account-stage')
+const goLiveStage = require('../../models/go-live-stage')
+
+module.exports = async (req, res) => {
+  const service = req.service
+  service.currentPspTestAccountStage = CREATED
+  try {
+    const pageData = {
+      requestForPspTestAccountNotStarted: (service.currentPspTestAccountStage === NOT_STARTED && service.currentGoLiveStage !== goLiveStage.LIVE),
+      requestForPspTestAccountSubmitted: service.currentPspTestAccountStage === REQUEST_SUBMITTED,
+      pspTestAccountCreated: service.currentPspTestAccountStage === CREATED,
+      isServiceLive: service.currentGoLiveStage === goLiveStage.LIVE
+    }
+
+    return response(req, res, 'request-psp-test-account/index', pageData)
+  } catch (error) {
+    return renderErrorView(req, res, false, error.errorCode)
```

Comment:
As this error handler doesn't do anything custom it should favour `next(error)`, the error will be handled in the same way but with consistent logging standards

(also wondering what error could actually happen here! I don't think anything above should throw, but not going to get in the way of more cautious rather than less)